### ISSUE TITLE
not enough memory

### ISSUE THREAD

**Issue by HR-RH** [vdemichev/DiaNN#790]
Hello, I have the following problems when using dia-nn 1.8.1. How should I solve them? thank you

RuntimeError: [enforce fail at ..\..\c10\core\CPUAllocator.cpp:76] data. DefaultCPUAllocator: not enough memory: you tried to allocate 23592960 bytes.

**Comment by vdemichev** [comment]
How does the log look like and what system do you run it on?

**Comment by HR-RH** [comment]
like this 
![image](https://github.com/vdemichev/DiaNN/assets/71167097/9176d674-6e14-4b9e-8b24-5dafe00ba0b2)

**Comment by HR-RH** [comment]
Excuse me, what does the system mean？

**Comment by HR-RH** [comment]
Do you mean the computer system?   windows

**Comment by vdemichev** [comment]
RAM amount, is it running on a virtual machine?

**Comment by HR-RH** [comment]
No, it runs on the local computer. Is there a solution? thank you

**Comment by HR-RH** [comment]
Hello, This problem still exists. How can I solve it? I still haven't found a solution. Is it because my computer has too little running memory? thanks

**Comment by HR-RH** [comment]
Hello, This problem still exists.  I still haven't found a solution. Please allow me to ask you one more time. How can I solve it? Is it because my computer has too little running memory? thanks

**Comment by vdemichev** [comment]
RAM amount?
